crates/sui-graphql-rpc/src/types/balance.rs

```diff
@@ -3,17 +3,15 @@
 
 use async_graphql::*;
 
-pub(crate) struct Balance;
-pub(crate) struct BalanceConnection;
+use super::big_int::BigInt;
 
-#[allow(unreachable_code)]
-#[allow(unused_variables)]
-#[Object]
-impl Balance {
-    async fn id(&self) -> ID {
-        unimplemented!()
-    }
+#[derive(Clone, Debug, PartialEq, Eq, SimpleObject)]
+pub(crate) struct Balance {
+    // pub(crate) coin_type: MoveType,
```

Comment:
nit: comment

Reply:
this is intentional as the impl is coming soon